packages/angular/READMEOSS.html

```
@@ -158,10 +158,10 @@
 				<td class="componentSummary"><a href="#compDetail_470898">LICENSE AND COPYRIGHT INFORMATION FOR COMPONENT @stencil/core (npm) - 4.41.3</a></td>
 			</tr>
 			<tr>
-				<td id="BlackDuckCompSummCellInsertionPoint" class="componentSummary">@stencil/core (npm) - 4.43.4</td>
+				<td id="BlackDuckCompSummCellInsertionPoint" class="componentSummary">@stencil/core (npm) - 4.43.5</td>
```

HTMLHint (1.9.2)

[error] The id value [ BlackDuckCompSummCellInsertionPoint ] must be unique.

(id-unique)